force-app/main/default/classes/ContactRollupsSchedulable.cls

```diff
@@ -7,66 +7,69 @@
  *
  */
 
-public class ContactRollupsSchedulable implements Database.Batchable<SObject>, Schedulable {
+//Stuff to do
+//1. See if we can pass this universe of IDs to the next batch.
+//2.
+
+public class ContactRollupsSchedulable implements Database.Batchable<SObject>, System.Schedulable, Database.Stateful {
+    @TestVisible
+    private QueryBuilder queryBuilder = new QueryBuilder();
     @TestVisible
-    private static AttendanceService attendanceService = new AttendanceService();
+    private Set<Id> contactIds = new Set<Id>();
 
+    // This kicks off this scheduable / batch class as the batch job
+    // Leaving the default of 200 because the Service Delivery Batch
+    // will need to find all child records and maintain the state of the
+    // contacts.
     public void execute(SchedulableContext context) {
-        Database.executeBatch(new ContactRollupsSchedulable());
+        Database.executeBatch(new ContactRollupsSchedulable(), 2000);
     }
 
+    // Find all of the contacts that have
```

Comment:
We could move this because we have a redundancy across batches but let's leave it here for now.